tree/treeplayer/inc/ROOT/TTreeProcessorMT.hxx

```diff
@@ -88,8 +89,9 @@ private:
    /// User-defined selection of entry numbers to be processed, empty if none was provided
    const TEntryList fEntryList; // const to be sure to avoid race conditions among TTreeViews
    const Internal::FriendInfo fFriendInfo;
+   ROOT::TThreadExecutor fPool; ///<! Thread pool for processing.
```

Comment:
Please add a comment that this data member should never be moved below the `TThrededObject`

Reply:
You'll get tons of test failures if you try :-) But yes, will do!